bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,304 @@
+At the end of the epoch,
+the validator BLS signatures are aggregated to create a compact checkpoint
+that is timestamped on the Bitcoin ledger.
+The BLS voting mechanism achieves a significant reduction in the cost of
+checkpoints, while the epoching mechanism specifies a defined frequency
+for checkpointing in the Bitcoin blockchain.
+
+### Create BLS Key
+
+To generate your BLS key, you'll need to use your validator address from the previous step. 
+Run this command:
+
+```shell
+babylond create-bls-key <address>
+```
+
+Replace `<address>` with your validator address from the earlier keyring 
+generation (it should look similar to `bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9`). 
+
+The system will automatically:
+1. Generate a new BLS key pair
+2. Associate it with your validator address
+3. Store it in your node's configuration file at 
+`~/.<path>/config/priv_validator_key.json`
+
+This key will be used automatically by your validator software when it needs 
+to participate in epoch-end signature collection. The BLS signatures help 
+create compact, efficient proofs of consensus that can be later timestamped to Bitcoin.
+
+Important: The `priv_validator_key.json` file contains sensitive key material. 
+Make sure to backup this file and store it securely, as it's essential for your 
+validator's operation and cannot be recovered if lost.
+
+## Prepare Validator Configuration
+
+Create your validator configuration file:
+
+```shell
+cat > <path>/config/validator.json << EOF
+{
+  "pubkey": $(cat pubkey.json),
+  "amount": "1000ubbn",
+  "moniker": "my-validator",
+  "commission-rate": "0.10",
+  "commission-max-rate": "0.20",
+  "commission-max-change-rate": "0.01",
+  "min-self-delegation": "1"
+}
+EOF
+```
+
+This command creates the configuration file with your validator settings:
+- `pubkey`: Your validator's public key
+- `amount`: Initial self-delegation amount
+- `moniker`: Your validator's name
+- `commission-rate`: Current commission rate
+- `commission-max-rate`: Maximum commission rate
+- `commission-max-change-rate`: Maximum daily commission change 
+- `min-self-delegation`: Minimum self-delegation amount
+
+> Note: The command will create the file if it doesn't exist. No need for a separate `touch` command.
+
+## Creating Validator
+
+Unlike traditional Cosmos SDK chains that use the `staking` module, 
+Babylon uses the `checkpointing` module for validator creation and management.
+
+The creation process requires your previously generated BLS key, 
+which should be located at `<path>/config/priv_validator_key.json`, 
+where `<path>` is the `--home` directory you specified when setting up your node.
+
+To create your validator, run the following command:
+
+```shell 
+babylond tx checkpointing create-validator \
+    ./<path>/config/validator.json \
+    --chain-id bbn-test-5 \
+    --gas "auto" \
+    --gas-adjustment 1.5 \
+    --gas-prices "0.005ubbn" \
+    --from <your-key-name>
+```
```

Comment:
Do we need to add the `--keyring-backend test` here?

Reply:
Resolved as the [key creation command](https://github.com/babylonlabs-io/networks/blob/35145f8c59faab7d5630cd18b2b29783c3d8e6c4/bbn-test-5/babylon-validators/README.md?plain=1#L38) doesn't use `--keyring-backend test` anymore.